packages/reaction-core/tests/jasmine/server/integration/cart.js

```diff
@@ -79,12 +79,13 @@ describe("cart methods", function () {
     describe("cart/removeFromCart", function () {
       it("should remove item from cart", function (done) {
         spyOn(ReactionCore.Collections.Cart, "update");
+        spyOn(Meteor, "userId").and.returnValue(userId);
         const currentCart = Factory.create("cart");
         const cartId = currentCart._id;
 
         expect(currentCart.items[0]).toBeDefined();
         for (let cartItem of currentCart.items) {
-          Meteor.call("cart/removeFromCart", cartId, cartItem);
+          Meteor.call("cart/removeFromCart", cartItem._id);
         }
 
         let modifiedCart = ReactionCore.Collections.Cart.find({
```

Comment:
I'm not actually sure this test is passing. Yes, I see it work on the client, but if you uncomment and console.log `modifiedCart ` here, you'll see that the items still exist.  This could be timing/spy test related, but it could also be the method is actually failing (since this test has no other confirmation that Meteor call worked (other than it "cart/removeCart" throwing an error).